lib/freedom_patches/net_http_header.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module NetHTTPHeaderPatch
```

Comment:
It probably works fine, but might be worth checking this works with our finaldestination::HTTP module

https://github.com/discourse/discourse/blob/d45c53ed1fd0252d0e6b6274fbc4d2501036c8eb/lib/final_destination/http.rb#L1-L1